### PR TITLE
Powertoys: added .reg files for PowerRename context menu item

### PR DESCRIPTION
**Pull request by ianharmon**
the .reg files' URLs will need to be updated before this PR gets merged

### CODE REVIEW

File: bucket/powertoys.json [hunk 1]
```diff
@@ -3,15 +3,49 @@
     "description": "A set of utilities for power users to tune and streamline their Windows experience for greater productivity.",
     "version": "0.13.0",
     "license": "MIT",
-    "url": "https://github.com/microsoft/PowerToys/releases/download/v0.13.0/PowerToysSetup.msi",
-    "hash": "08361ae389467e9bb6180a5b28dc11a647e8ffb5ade129d9b8894324ec450a03",
+    "url": [
+        "https://github.com/microsoft/PowerToys/releases/download/v0.13.0/PowerToysSetup.msi",
+        "https://raw.githubusercontent.com/ianharmon/scoop-extras/powertoys/scripts/powertoys/powertoys-install-context.reg",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/powertoys/install-context.reg",
```

File: bucket/powertoys.json [hunk 2]
```diff
@@ -3,15 +3,49 @@
...
+        "https://raw.githubusercontent.com/ianharmon/scoop-extras/powertoys/scripts/powertoys/powertoys-uninstall-context.reg"
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/powertoys/uninstall-context.reg"
```

File: scripts/powertoys/powertoys-install-context.reg [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Windows Registry Editor Version 5.00
```
Comment (Ash258) [inline scripts/powertoys/powertoys-install-context.reg]: Do not add nested name.

`git mv scripts/powertoys/powertoys-install-context.reg scripts/powertoys/install-context.reg`
`git mv scripts/powertoys/powertoys-uninstall-context.reg scripts/powertoys/uninstall-context.reg`

File: bucket/powertoys.json [hunk 4]
```diff
@@ -3,15 +3,49 @@
...
     "extract_dir": "PowerToys",
     "shortcuts": [
         [
             "PowerToys.exe",
             "PowerToys"
         ]
     ],
+    "notes": [
+        "Add the PowerRename tool as a context menu item by running:",
+        "\"$dir\\powertoys-install-context.reg\"",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
```

File: bucket/powertoys.json [hunk 5]
```diff
@@ -3,15 +3,49 @@
...
+    ],
     "extract_dir": "PowerToys",
     "shortcuts": [
         [
             "PowerToys.exe",
             "PowerToys"
         ]
     ],
+    "notes": [
+        "Add the PowerRename tool as a context menu item by running:",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "Add the PowerRename tool as a context menu item by running: '$dir\\powertoys-install-context.reg'",
```

File: bucket/powertoys.json [hunk 6]
```diff
@@ -3,15 +3,49 @@
...
+        "\"$dir\\powertoys-install-context.reg\"",
+        "If installed globally, other users can enable the context menu item in the PowerToys app.",
+        "",
+        "To remove the context menu item, which should be done before uninstalling:",
+        "\"$dir\\powertoys-uninstall-context.reg\"",
+        "You may also need to close Explorer or other file managers when uninstalling."
```
Comment (Ash258) [inline bucket/powertoys.json]: Why??

File: bucket/powertoys.json [hunk 7]
```diff
@@ -3,15 +3,49 @@
...
             "PowerToys.exe",
             "PowerToys"
         ]
     ],
+    "notes": [
+        "Add the PowerRename tool as a context menu item by running:",
+        "\"$dir\\powertoys-install-context.reg\"",
+        "If installed globally, other users can enable the context menu item in the PowerToys app.",
+        "",
+        "To remove the context menu item, which should be done before uninstalling:",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "To remove the context menu item, which should be done before uninstalling: '$dir\\powertoys-uninstall-context.reg'",
```

File: bucket/powertoys.json [hunk 8]
```diff
@@ -3,15 +3,49 @@
...
+        "\"$dir\\powertoys-uninstall-context.reg\"",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
```

File: bucket/powertoys.json [hunk 9]
```diff
@@ -3,15 +3,49 @@
...
+        "You may also need to close Explorer or other file managers when uninstalling."
+    ],
+    "post_install": [
+        "if (Test-Path \"$dir\\powertoys-install-context.reg\") {",
+        "  $content = Get-Content \"$dir\\powertoys-install-context.reg\"",
+        "  $content = $content.Replace('$installdir', $dir.Replace('\\', '\\\\'))",
+        "  if ($global) {",
+        "    $content = $content.Replace('HKEY_CURRENT_USER\\Software\\Classes', 'HKEY_CLASSES_ROOT')",
+        "  }",
+        "  $content | Set-Content -Path \"$dir\\powertoys-install-context.reg\"",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "  Set-Content \"$dir\\powertoys-install-context.reg\" $content -Encoding Ascii",
```

File: bucket/powertoys.json [hunk 10]
```diff
@@ -3,15 +3,49 @@
...
+        "}",
+        "if (Test-Path \"$dir\\powertoys-uninstall-context.reg\") {",
+        "  if ($global) {",
+        "    $content = Get-Content \"$dir\\powertoys-uninstall-context.reg\"",
+        "    $content = $content.Replace('HKEY_CURRENT_USER\\Software\\Classes', 'HKEY_CLASSES_ROOT')",
+        "    $content | Set-Content -Path \"$dir\\powertoys-uninstall-context.reg\"",
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "    Set-Content \"$dir\\powertoys-uninstall-context.reg\" $content -Encoding Ascii",
```

**Comment by JanPokorny** [comment]
Looks like the issue still persists, why was this closed?

**Pull request closed without merging**